features/providers/civil_application_journey.feature

```diff
@@ -361,7 +361,7 @@ Feature: Civil application journeys
     Then I click 'Save and continue'
     Then I should be on a page showing 'Check your answers'
 
-  @javascript @vcr
+  @javascript @vcr @alex
```

Comment:
🤔

Reply:
Whoops. And this is why we code review! 😂